connect/file/src/main/java/org/apache/kafka/connect/file/FileStreamSinkConnector.java

```diff
@@ -66,4 +70,14 @@ public void start(Map<String, String> props) {
     public void stop() {
         // Nothing to do since FileStreamSinkConnector has no background monitoring.
     }
+
+    @Override
+    public ConfigDef defineConfig() {
+        if (this.configDef != null) {
```

Comment:
@Ishiihara Not sure you've addressed this? Seems like it's easier to drop the `configDef` field in `Connector` entirely and avoid this?